src/to-have-style.js

```diff
@@ -1,6 +1,7 @@
 import { matcherHint } from 'jest-matcher-utils';
 import { diff } from 'jest-diff';
 import chalk from 'chalk';
+import { StyleSheet } from 'react-native';
```

Comment:
nit: pls move `react-native` as a first import

Reply:
Done, do we have any rule for those? we could add [import/order](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) to ESLint config